### PR TITLE
add angular-bootstrap-switch v0.5.0

### LINKED ISSUE #7597: [Request] Add angular-bootstrap-switch
**Library name:** angular-bootstrap-switch
**Git repository url:** https://github.com/frapontillo/angular-bootstrap-switch
**npm package url(optional):** https://npmjs.org/package/angular-bootstrap-switch
**License(s):** Apache License, Version 2.0

### PR DESCRIPTION
**Pull request by dannyxx001**
PR for #7597 
@pvnr0082t please help me review this PR, thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/frapontillo/angular-bootstrap-switch
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/angular-bootstrap-switch
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 228
   - Count of watchers: 17
   - Count of forks: 99
 * NPM download stats (optional):
   - Downloads in the last day: 38
   - Downloads in the last week: 205
   - Downloads in the last month: 670

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

with autoupdate config

close #7597, cc @frapontillo

### CODE REVIEW

**Comment by frapontillo** [comment]
@dannyxx001 good for me! Thanks!

File: ajax/libs/angular-bootstrap-switch/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "name": "Francesco Pontillo",
+    "email": "francescopontillo@gmail.com",
+    "url": "https://github.com/frapontillo"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/frapontillo/angular-bootstrap-switch.git"
+  },
+  "license": "Apache-2.0",
+  "description": "angular-bootstrap-switch ========================",
```
Comment (pvnr0082t) [inline ajax/libs/angular-bootstrap-switch/package.json]: Please use something meaningful here, maybe the description on GitHub will be better.

Comment (dannyxx001) [inline ajax/libs/angular-bootstrap-switch/package.json]: OK, I had updated it. Please review again.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this pull request is ok. Please review it again, thank you.

**Comment by PeterDaveHello** [comment]
Thank you all.

**Pull request merged**